kornia/contrib/connected_components.py

```diff
@@ -1,7 +1,10 @@
 import torch
 import torch.nn.functional as F
 
+from kornia.utils.image import perform_keep_shape_image
 
+
+@perform_keep_shape_image
```

Comment:
@justanhduc seems that with the decorator, some functional symbols cannot be traced

Reply:
It is because of the `kwargs`. Do you know any workaround?

Reply:
not right now - but I'll do a bit of research on that.